unit_test/tstGridOperator3d.hpp

```diff
@@ -178,9 +178,9 @@ void gatherScatterTest()
     int minimum_halo_size = 0;
 
     // Make mesh.
-    auto mesh =
-        createUniformMesh( TEST_MEMSPACE(), parser.propertyTree(), global_box,
-                           minimum_halo_size, MPI_COMM_WORLD );
+    auto mesh = createUniformMesh( TEST_MEMSPACE(), parser.propertyTree(),
```

Comment:
Should we add a creation function for adaptive?

Reply:
done